samples/notes/android/app/src/main/kotlin/org/mobilenativefoundation/store/notes/app/wiring/AppModule.kt

```diff
@@ -0,0 +1,93 @@
+package org.mobilenativefoundation.store.notes.app.wiring
+
+import android.app.Application
+import android.content.Context
+import com.dropbox.external.store5.Bookkeeper
+import com.dropbox.external.store5.Market
+import com.dropbox.external.store5.Store
+import com.dropbox.external.store5.impl.MemoryLruCache
+import com.squareup.anvil.annotations.ContributesTo
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlDriver
+import dagger.Module
+import dagger.Provides
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.mobilenativefoundation.store.notes.android.app.NotesDatabase
+import org.mobilenativefoundation.store.notes.android.common.api.Note
+import org.mobilenativefoundation.store.notes.android.common.scoping.AppScope
+import org.mobilenativefoundation.store.notes.android.common.scoping.SingleIn
+import org.mobilenativefoundation.store.notes.app.extension.tryDeleteAllFailed
+import org.mobilenativefoundation.store.notes.app.extension.tryDeleteAllNotes
+import org.mobilenativefoundation.store.notes.app.extension.tryDeleteFailed
+import org.mobilenativefoundation.store.notes.app.extension.tryDeleteNote
+import org.mobilenativefoundation.store.notes.app.extension.tryGetFailed
+import org.mobilenativefoundation.store.notes.app.extension.tryGetNote
+import org.mobilenativefoundation.store.notes.app.extension.tryWriteFailed
+import org.mobilenativefoundation.store.notes.app.extension.tryWriteNote
+import org.mobilenativefoundation.store.notes.app.market.Key
+import javax.inject.Named
+
+@Module
+@ContributesTo(AppScope::class)
+object AppModule {
+    @Provides
+    @SingleIn(AppScope::class)
+    fun provideContext(application: Application): Context = application.applicationContext
```

Comment:
what is binding the application into the app scope? I noticed the dagger component doesn't have a binds instance

Reply:
👍🏽